src/pudl/analysis/record_linkage/eia_ferc1_splink_rule_definitions.py

```diff
@@ -0,0 +1,87 @@
+import splink.duckdb.comparison_level_library as cll
+import splink.duckdb.comparison_library as cl
+import splink.duckdb.comparison_template_library as ctl
+from splink.duckdb.blocking_rule_library import block_on
+
+blocking_rule_1 = "l.report_year = r.report_year and jaro_winkler_similarity(l.plant_name, r.plant_name) >= .7"
```

Comment:
This filter condition could have [negative performance consequences](https://moj-analytical-services.github.io/splink/topic_guides/blocking/performance.html#efficient-blocking-rules).   

(it's not a problem if it's running fast enough for your needs)

Two options for improving performance whilst retaining a 'similar' intention could be:
replacing 
`jaro_winkler_similarity(l.plant_name, r.plant_name) >= .7`
with  a substring like
`substr(l.plant_name,1,2) = substr(r.plant_name,1,2)`

or possibly pre-processing the input data to take a soundex/dmetaphone of `plant_name` and blocking on dmeta_plan_name

Reply:
Got it! I think replacing these `jaro_winkler_similarity` blocking rules makes sense.